job_templates/ci_launcher_job.xml.em

```diff
@@ -2,7 +2,7 @@
 <project>
   <actions/>
   <description>
-    Note: The Windows jobs ignore the middleware selection flags.
+    Launches a build of the CI job for each supported platform with the same set of parameters.
```

Comment:
while we'll have to maintain this, I think it's good to document it: https://github.com/ros2/ci/pull/96/commits/4b819d4c044b335e138d957178df706a9c5ec8d3